appveyor.yml

```diff
@@ -65,8 +65,8 @@ test_script:
     # Use the Agg backend in Matplotlib
   -  echo backend:Agg > matplotlibrc
 
-  # Run unit tests with nose
-  - "python -c \"import nose; nose.main()\" -v -s skimage"
+  # Run unit tests with pytest
+  - "python -c \"import pytest; pytest.main(['skimage'])\""
```

Comment:
I would be very happy to use the pytest script. I kept this as close as possible to the previous patch, but I'll fix this later on.